### DISCUSSION TITLE
overhangs question

### DISCUSSION THREAD

**Discussion by Edzi11** [OrcaSlicer/OrcaSlicer#949] [Q&A]
I see that there are two overhang options but im confused as to what the classic option is for. i can enable both but im not sure if i should do that

**Comment by tome9111991** [comment]
@Edzi11 
The Classic mode use full speed until overhang.
![classic](https://user-images.githubusercontent.com/57866234/236813469-194be50b-7c6f-4ab7-934d-442f5c97f22b.png)

the none Classic mode will slow down the speed before overhang beginn. (is new PS behavior)
on peak from overhang you can reach overhang speed.
![none classic](https://user-images.githubusercontent.com/57866234/236813813-f770cd65-011b-40d0-911d-b059fcf4d865.png)
